pepper-apis/src/test/java/org/broadinstitute/ddp/route/ServerInSameProcessIntegrationTestSuite.java

```diff
@@ -9,15 +9,18 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
+        AdminCreateStudyParticipantRouteTest.class,
         GetParticipantInfoRouteTest.class,
         ListCancersRouteTest.class,
         GetDsmDrugSuggestionsRouteTest.class,
+        GovernedParticipantRegistrationRouteTest.class,
         PatchFormAnswersRouteStandaloneTest.class,
         PutFormAnswersRouteStandaloneTest.class,
         PutFormAnswersRouteStandalone2Test.class,
         PutFormAnswersRouteStandalone3Test.class,
         GetActivityInstanceRouteStandaloneTest.class,
-        UserActivityInstanceListRouteStandaloneTest.class
+        UserActivityInstanceListRouteStandaloneTest.class,
+        UserRegistrationRouteTest.class
```

Comment:
These tests now look at mock objects, so we need the server to be in the same process as these tests, so I've moved them here.